app/services/action_create_service.rb

```diff
@@ -75,21 +75,26 @@ def deny(comments)
     unless request.state == Request::NOTIFIED_STATE
       raise Exceptions::InvalidStateTransitionError, "Current request is not in notified state"
     end
-    raise Exceptions::ApprovalError, "Reason to deny the request is missing" unless comments
+    raise Exceptions::UserError, "Reason to deny the request is missing" unless comments
```

Comment:
@bzwei This error message is a bit confusing, is it saying that the Approver didn't give a reason why he denied the request.
Can the message be "A reason has to be provided if a request is being denied" or
"The approver has not specified a reason about why a request has been denied"